src/LieTheory/WeylGroup.jl

```diff
@@ -75,11 +75,56 @@ Construct a Weyl group element from the given word.
 The word must be a list of integers, where each integer is the index of a simple reflection.
 
 If the word is known to be in short lex normal form, the normalization can be skipped by setting `normalize=false`.
+
+# Examples
+```jldoctest
+julia> W = weyl_group(:A, 2);  wrd = [1, 2, 1];
+
+julia> x = W(wrd)
+s1 * s2 * s1
+
+julia> word(x) == wrd
+true
```

Comment:
```suggestion
julia> W = weyl_group(:A, 2);

julia> x = W([1, 2, 1])
s1 * s2 * s1
```
I think these two lines may be a bit misleading. This only works here because `wrd` is already in short lex normal form.

We don't guarantee that `word(W(wrd)) == wrd` in general (only for words in short lex normal form), but we do guarantee `W(word(x)) == x` in general.

And I don't quite like how much vertical space these examples take in the documentation. A little shortened form without the misleading checks IMO is enough for examples, the guarantee should be checked in proper tests instead.

Reply:
That makes sense, I was unsure how to arrange this without a misleading example but this approach is better. Thanks!